treeherder/config/settings.py

```diff
@@ -48,6 +48,10 @@
 MEDIA_ROOT = path("media")
 MEDIA_URL = "/media/"
 
+# Default minimum regression threshold for perfherder is 1% (otherwise
+# e.g. the build size tests will alert on every commit)
+PERFHERDER_REGRESSION_THRESHOLD = 1
```

Comment:
we should make this 2% to match graph server;  also in graph server we have 10% for the dromaeo threshold- we might want to consider having exception lists.